.buildkite/custom-tests.json

```diff
@@ -0,0 +1,38 @@
+{
+    "tests": [
+        {
+            "test_name": "build-fam-gnu",
+            "command": "cargo build --release --features=fam-wrappers",
+            "platform": [
+                "x86_64",
+                "aarch64",
+                "riscv64"
+            ]
+        },
+        {
+            "test_name": "build-fam-musl",
+            "command": "cargo build --release --features=fam-wrappers --target {target_platform}-unknown-linux-musl",
+            "platform": [
+                "x86_64",
+                "aarch64"
+            ]
+        },
+        {
+            "test_name": "build-serde-gnu",
+            "command": "cargo build --release --features=serde",
+            "platform": [
+                "x86_64",
+                "aarch64",
+                "riscv64"
+            ]
+        },
+        {
+            "test_name": "build-serde-musl",
+            "command": "cargo build --release --features=serde --target {target_platform}-unknown-linux-musl",
+            "platform": [
+                "x86_64",
+                "aarch64"
+            ]
+        }
+    ]
+}
```

Comment:
These don't yet show up in buildkite because the pipeline steps don't include `./rust-vmm-ci/.buildkite/autogenerate_pipeline.py -t .buildkite/custom-tests.json | buildkite-agent pipeline upload`. Let me temporarily add it, retrigger the pipeline here. If it works, I'll remove it again and only permanently add it once this is merged (as otherwise we'll break CI on all other PRs)

Reply:
Seems to all work: https://buildkite.com/rust-vmm/kvm-ioctls-ci/builds/1250#_ :tada:

Reply:
Commented out the steps in the pipeline definition again until after we merge this